packages/react-core/src/components/ContextSelector/ContextSelector.tsx

```diff
@@ -37,8 +37,10 @@ export interface ContextSelectorProps extends Omit<ToggleMenuBaseProps, 'menuApp
   isFullHeight?: boolean;
   /** Labels the Context Selector for Screen Readers */
   screenReaderLabel?: string;
-  /** Text that appears in the Context Selector Toggle */
-  toggleText?: string;
+  /** Component or string that will be used in the Context Selector.
```

Comment:
Can this be sentence case please. "Component or string that will be used in the context selector."

Reply:
Done ✅